scim-server/scim-server-common/src/test/java/org/apache/directory/scim/server/provider/UpdateRequestTest.java

```diff
@@ -27,23 +33,7 @@
 
 import javax.enterprise.inject.Instance;
 
-import jdk.nashorn.internal.ir.annotations.Ignore;
```

Comment:
This was not the correct `@Ignore` annotation for unit tests and caused a "Not API" warning. JUnit 5 moved to `@Disabled`.